chain-core/src/init/address.rs

```diff
@@ -135,6 +169,58 @@ impl RedeemAddress {
     }
 }
 
+impl CroAddress<RedeemAddress> for RedeemAddress {
```

Comment:
As mentioned her: https://github.com/crypto-com/chain/pull/187/files#r302037650 RedeemAddress could be hex-only for start

Reply:
ok
i'll modify as follow
RedeemAddress: hex only
ExtendedAddress: bech32 only